src/ocrd/mets_server.py

```diff
@@ -578,10 +607,10 @@ async def add_file(
         uvicorn_kwargs['log_config'] = None
         uvicorn_kwargs['access_log'] = False
 
-        self.log.debug("Starting uvicorn")
+        self.log.info("Starting the uvicorn Mets Server")
         uvicorn.run(app, **uvicorn_kwargs)
 
-
+# TODO: Not required after #1284, consider removing
```

Comment:
Cannot propose outside the diff context, but if we don't need it any more, feel free to rremove `def is_socket_in_use...`.

Reply:
Resolved with [5055309](https://github.com/OCR-D/core/pull/1284/commits/50553093180ac6b641273c08d559bbadf5e9b1d2)